File: ts/wam/3806.ts
export const WamPnhDaily = 3806 as const

export interface PnhDailyEvent {
  /** field 1, wire `community_id` */
  communityId?: string
  /** field 2, wire `pnh_indicator_clicks_chat` */
  pnhIndicatorClicksChat?: number
  /** field 3, wire `pnh_indicator_clicks_info_screen` */
  pnhIndicatorClicksInfoScreen?: number
  /** field 4, wire `reaction_delete_count` */
  reactionDeleteCount?: number
  /** field 5, wire `reaction_open_tray_count` */
  reactionOpenTrayCount?: number
  /** field 6, wire `type_of_group` */
  typeOfGroup?: TYPEOFGROUPENUM
  /** field 7, wire `mapping_missing` */
  mappingMissing?: number
  /** field 8, wire `total_contacts` */
  totalContacts?: number
}

File: go/wam/3806.go
const WamPnhDaily = 3806 // channel: regular

type PnhDailyEvent struct {
	CommunityId string `wam:"community_id"` // field 1
	PnhIndicatorClicksChat int64 `wam:"pnh_indicator_clicks_chat"` // field 2
	PnhIndicatorClicksInfoScreen int64 `wam:"pnh_indicator_clicks_info_screen"` // field 3
	ReactionDeleteCount int64 `wam:"reaction_delete_count"` // field 4
	ReactionOpenTrayCount int64 `wam:"reaction_open_tray_count"` // field 5
	TypeOfGroup TYPEOFGROUPENUM `wam:"type_of_group"` // field 6
	MappingMissing int64 `wam:"mapping_missing"` // field 7
	TotalContacts int64 `wam:"total_contacts"` // field 8
}